application/frontend/controllers/PasswordController.php

```diff
@@ -74,4 +73,46 @@ public function actionUpdate()
 
         return $pwMeta;
     }
+
+    /**
+     * Assess whether a password will pass validation checks without actually saving the password.
+     * @throws BadRequestHttpException
+     */
+    public function actionAssess()
+    {
+        /** @var string $newPassword */
+        $newPassword = $this->getPasswordFromRequestBody();
+
+        /** @var User $user */
+        $user = \Yii::$app->user->identity;
+
+        $testPassword = Password::create($user, $newPassword);
+
+        if (! $testPassword->validate('password')) {
+            $errors = join(', ', $testPassword->getErrors('password'));
+            \Yii::warning([
+                'action' => 'password/assess',
+                'status' => 'error',
+                'employee_id' => $user->employee_id,
```

Comment:
```suggestion
                'email' => $user->email,
```
Can we change this to email to make searching logs easier?